sources/osgUtil/Composer.js

```diff
@@ -113,22 +123,51 @@ define( [
             return false;
         },
 
+        /* develblock:start */
+        // debug only check for specific bad condition:
+        // in webgl 1.0 you cannot read and write on the same texture
+        // so you shouldn't bind a FOB texture as input and output
+        debugCheckRttNotReadWrite: function ( textureResult, stateSet ) {
+
+            for ( var k = 0, lt = stateSet.getNumTextureAttributeLists(); k < lt; k++ ) {
+                var textureBinded = stateSet.getTextureAttribute( k, 'Texture' );
+                Notify.assert( textureResult !== textureBinded, 'Composer: write/read at the same time on a texture is undefined behavior' );
+
+            }
+        },
+        /* develblock:end */
+
         build: function () {
```

Comment:
Is it possible to add an option to eslint to disallow functions longer than 100 lines? :)